action.yml

```diff
@@ -68,9 +83,17 @@ runs:
       id: create-branch
       shell: bash
       run: |
+        # validate and convert annotation to map[string]string
+        if [[ -n '${{ inputs.annotation }}' ]]; then
+          annotation=$(echo '${{ inputs.annotation }}' | jq -cr \
```

Comment:
not sure what is this jq command for, can you add a comment please

Reply:
Done, could you also validate if the command does right things from your perspective (having the comment).

Reply:
to be honest I would avoid any kind of complex logic here, it's been super hard to support these actions, I really regret going with bash instead of javascript.
Can we just pass the data as it is to the neonctl?

Reply:
Done. I left only `jq -c` (--compact) - which will the annotation json to a single line (otherwise, actions breaks if annotation is a multiline JSON)

Reply:
In the team we agreed to extract annotation values without modifying the action interface.
So now there is a different jq operators with the comments.